uint8 order
std_msgs/Header header
float32 period
float32[] x_params
float32[] y_params
float32[] z_params
